.github/workflows/tag-release.yaml

```diff
@@ -37,7 +37,12 @@ jobs:
       - name: Update Tag
         uses: actions/github-script@v7.0.1
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # use a real access token instead of GITHUB_TOKEN default.
+          # required so that the results of this tag creation can trigger the build workflow
+          # https://stackoverflow.com/a/71372524
+          # https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow
+          # this token will need to be renewed anually in January
```

Comment:
I suggest that you store a companion secret encoding the upcoming expiration date of `LL_TAG_RELEASE_TOKEN` and, once it gets close, start emitting a [warning message](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-warning-message) on the GH workflow page. You don't need to use the extra `::warning ...` syntax; it works to code:
`echo "::warning::LL_TAG_RELEASE_TOKEN must be recreated before $expiration"`

Reply:
capturead as #3386